src/PrestaShopBundle/Resources/views/Admin/Product/CatalogPage/Lists/products_table.html.twig

```diff
@@ -129,7 +132,7 @@
               'value': filter_column_price,
             } %}
           </th>
-
+          <th class="text-center"></th>
```

Comment:
I guess this line is not needed here, as you do not add values in the array in #11483

Reply:
Found out this was used for the filter inputs.